### PR TITLE
Sa 95

### PR DESCRIPTION
**Pull request by tathibo1**
https://sparkpost.atlassian.net/browse/SA-95

 A/C:
- Our API Docs accurately reflect the Metrics API features
- Existing support docs accurately reflect the Metrics API features
- The scope is any feature that is shipped to production

Changes made:
- added missing templates list endpoint
- documents that day precision may be used for aggregate requests
- documents that day+ precisions will use UTC regardless of timezone QP
- updated advanced comparator schema query to display URL and a truncated example response
- clarifies the difference in terminology between the Deliverability Add-On product and deliverability as a general term.

Changes not made:
- Uniques are still handled with raw data - we don’t need to explain that uniques are per rollup precision or that 5/15min or week/month are aggregates of 1min/day because it makes no difference to the API consumer.
- We don’t really want people using 15min precision at 48 hours for performance reasons, so I’m going to skip adding more documentation around that.

### CODE REVIEW

**Review by gwalls — CHANGES_REQUESTED** [review]

File: content/api/metrics.apib [hunk 1]
```diff
@@ -247,6 +243,49 @@ Compressed (whitespace and line end characters removed) and URL encoded:
 ```
 ?query_filters=%7B%22groupings%22%3A%5B%7B%22AND%22%3A%7B%22domains%22%3A%7B%22eq%22%3A%5B%22gmail.com%22%2C%22yahoo.com%22%2C%22hotmail.com%22%5D%2C%22like%22%3A%5B%22mail%22%5D%7D%2C%22sending_domains%22%3A%7B%22notEq%22%3A%5B%22sparkpost.com%22%5D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22templates%22%3A%7B%22eq%22%3A%5B%22default%22%5D%7D%2C%22campaigns%22%3A%7B%22like%22%3A%5B%22333%22%2C%22344%22%2C%22355%22%5D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22campaigns%22%3A%7B%22notLike%22%3A%5B%22Friday%22%5D%2C%22notEq%22%3A%5B%22stuff%22%2C%22things%22%5D%7D%7D%7D%5D%7D
 ```
+Multiple compare groups are evaluated such that the relationship between them is an AND operation.
+
+### Advanced Query JSON Schema [GET /v1/metrics/deliverability/query-filters-schema]
```
[the remark this answers is not in the document]
Comment (tathibo1) [reply content/api/metrics.apib:249]: Thanks for catching that, I've updated it and it's working for me now.

File: content/api/metrics.apib [hunk 2]
```diff
@@ -247,6 +243,49 @@ Compressed (whitespace and line end characters removed) and URL encoded:
 ```
 ?query_filters=%7B%22groupings%22%3A%5B%7B%22AND%22%3A%7B%22domains%22%3A%7B%22eq%22%3A%5B%22gmail.com%22%2C%22yahoo.com%22%2C%22hotmail.com%22%5D%2C%22like%22%3A%5B%22mail%22%5D%7D%2C%22sending_domains%22%3A%7B%22notEq%22%3A%5B%22sparkpost.com%22%5D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22templates%22%3A%7B%22eq%22%3A%5B%22default%22%5D%7D%2C%22campaigns%22%3A%7B%22like%22%3A%5B%22333%22%2C%22344%22%2C%22355%22%5D%7D%7D%7D%2C%7B%22OR%22%3A%7B%22campaigns%22%3A%7B%22notLike%22%3A%5B%22Friday%22%5D%2C%22notEq%22%3A%5B%22stuff%22%2C%22things%22%5D%7D%7D%7D%5D%7D
 ```
+Multiple compare groups are evaluated such that the relationship between them is an AND operation.
```
[the remark this answers is not in the document]
Comment (tathibo1) [reply content/api/metrics.apib]: I've moved it to the "Grouping Structure" section, after the example structure. Let me know if you think it seems more clear to put it prior to that.

File: content/api/metrics.apib [hunk 3]
```diff
@@ -106,14 +102,14 @@ The Metrics API uses [JSON Schema](https://json-schema.org/) to define and valid
...
-See [Advanced Query JSON Schema](#header-advanced-query-json-schema).
+See [Advanced Query JSON Schema](#metrics-get-advanced-query-json-schema).
 
 Please note, advanced filters cannot be used in conjunction with simple filters. Simple filters will be ignored if the `query_filters` parameter is used. 
 e.g. in `?domains=mailing.com&query_filters={}`, the domains query parameter will be ignored. 
 Other query parameters like `from`, `to`, `metrics`, `limit`, etc., will be processed as expected.
 
 #### Groupings Structure 
-The JSON object consists of a top level "groupings" key that has an array of compare objects
+The JSON object consists of a top level "groupings" key that has an array of compare objects.
```
Comment (gwalls) [inline content/api/metrics.apib:112]: I think I meant "root level" when I wrote this 😆 
```suggestion
The JSON object consists of a root level "groupings" array containing compare objects.
```

**Review by gwalls — APPROVED** [review]
Left one last suggestion, but otherwise lgtm. 

I think the description of the compare groups would look better if it was before the example, but either way is probably fine.

File: content/api/metrics.apib (hunk 3 above)
Comment (tathibo1) [reply content/api/metrics.apib]: The meaning is probably clear either way but I've made the suggested change.

**Pull request merged**